src/styles/blocks/card.css

```diff
@@ -0,0 +1,76 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  width: 200px;
+  border: 1px solid var(--bg-color-main);
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 37px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main);
+  margin: 0 var(--min-section-gap) 4px;
+}
+
+.card__code {
+  line-height: 14px;
+  color: var(--color-secondary);
+  margin-bottom: 16px;
+  margin-left: var(--min-section-gap);
+
+}
```

Comment:
due to your index.html - card__code is a block. It is not good idea to add external styles (positioning or margins) to BEM-blocks.
Checklist 7